src/ralph/data_center/apps.py

```diff
@@ -0,0 +1,9 @@
+from ralph.apps import RalphAppConfig
+
+
+class DataCenterConfig(RalphAppConfig):
+    name = 'ralph.data_center'
+    verbose_name = 'Data Center Assets'
+
+    def get_proxies(self):
+        return ['DCHost']
```

Comment:
ok. so @ar4s what do you think about making it working automatically for all proxy models?

We now have 7 proxy models:
```
<class 'ralph.operations.models.Change'>
<class 'ralph.operations.models.Incident'>
<class 'ralph.operations.models.Problem'>
<class 'ralph.operations.models.Failure'>
<class 'ralph.deployment.models.Deployment'>
<class 'ralph.data_center.models.hosts.DCHost'>
<class 'ralph.dhcp.models.DHCPEntry'>
```